pkg/controller/kogitoapp/kogitoapp_controller.go

```diff
@@ -53,6 +54,11 @@ import (
 
 var log = logger.GetLogger("controller_kogitoapp")
 
+const (
+	// imageStreamNamespace default namespace for the ImageStreams
+	imageStreamNamespace = "openshift"
```

Comment:
Isn't this same env declared somewhere? I do remember to see it but nor sure where exactly

Reply:
This will be removed.